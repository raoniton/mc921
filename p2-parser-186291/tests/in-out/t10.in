class Main {
    public static void main(String[] args) {
        int i, j;
        j = i + 1;
    }
}