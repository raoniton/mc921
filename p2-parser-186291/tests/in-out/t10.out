Program:
    ClassDecl: ID(name=Main) @ 1:1
        MainMethodDecl: @ 2:5
            ID: args @ 2:38
            Compound: @ 2:44
                VarDecl: ID(name=i) @ 3:13
                    Type: int @ 3:9
                VarDecl: ID(name=j) @ 3:16
                    Type: int @ 3:9
                Assignment: = @ 4:9
                    ID: j @ 4:9
                    BinaryOp: + @ 4:13
                        ID: i @ 4:13
                        Constant: int, 1 @ 4:17
